### PR TITLE
gsoc blog: purna

### PR DESCRIPTION
**Pull request by reshamas**

### CODE REVIEW

**Comment by reshamas** [comment]
[@purna135](https://github.com/purna135), please review:
https://pymcio--42.org.readthedocs.build/welcome.html

**Review by cluhmann — CHANGES_REQUESTED** [review]
Picture seems much larger than needed.  @OriolAbril can weigh in about whether it's possible to resize using the markdown style image links or whether it requires an image directive.  Other than that, it looks good to me.

File: blog/blog_gsoc_2022.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Increase Support for Batched Multivariate Distributions
+
+**Bio**
+>I am a final-year student at Hyderabad Central University pursuing a Master's in Computer Applications. I've been working in the fields of data science and machine learning for the past year. I enjoy working on complex problems and am a technical instructor. In my spare time, I enjoy contributing what I've learned to open-source projects.
+
+**Connecting** 
+- Twitter: [@purna_mansingh](https://twitter.com/purna_mansingh)
+- LinkedIn: [purna135](https://www.linkedin.com/in/purna135/)
+- GitHub: [@purna135](http://github.com/purna135)
```
Comment (OriolAbril) [inline blog/blog_gsoc_2022.md:28]:
```suggestion
**Project Name**

Increase Support for Batched Multivariate Distributions

::::{grid}
:::{grid-item} 
:columns: 12 6 6 4

![photo of Purna Chandra Mansingh](../_static/gsoc_2022/purna.jpg)
:::
:::{grid-item}
:columns: 12 6 6 8

**Bio**
>I am a final-year student at Hyderabad Central University pursuing a Master's in Computer Applications. I've been working in the fields of data science and machine learning for the past year. I enjoy working on complex problems and am a technical instructor. In my spare time, I enjoy contributing what I've learned to open-source projects.

**Connecting** 
- Twitter: [@purna_mansingh](https://twitter.com/purna_mansingh)
- LinkedIn: [purna135](https://www.linkedin.com/in/purna135/)
- GitHub: [@purna135](http://github.com/purna135)
:::
::::
```

**Comment by OriolAbril** [comment]
How does this look?

The 4 numbers config indicates the width on different screen sizes. It uses a system of 12 columns total. Therefore, for small screens, both items are full width, the image comes before, then the text. For medium and largish screens, the width is divided half-half, for big screens, the text is twice as wide as the image.

We can change the order and or move content in and out of the grid elements as needed.

---

big screen:
image

medium screen (ipad simulated on firefox dev tools)
image

small screen (iphone se simulated of firefox dev tools)
image

File: blog/blog_gsoc_2022.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+:::
+::::
+
+Learn more at [Purna's GSOC blog](https://purna135.github.io/gsoc/gsoc-2022-with-pymc/).
+
+1. __What motivated you to apply for GSoC with PyMC?__
+    GSoC is a place where I can not only apply my existing skills but also learn new ones. And the learning does not stop with technical knowledge. GSoC introduces me to a new paradigm for collaboratively developing code. Furthermore, GSoC is a platform that allows me to build on and hone my current skills, which motivates me to apply for GSoC.
```
[the remark this answers is not in the document]
Comment (reshamas) [reply blog/blog_gsoc_2022.md:42]: Adding a single line break for question 1

File: blog/blog_gsoc_2022.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+We are excited to introduce to the community our cohort of GSoC students working on PyMC, Aesara and ArViZ.
+:::
+
+## Purna Chandra Mansingh
+
+**Project Name**
+
+Increase Support for Batched Multivariate Distributions
+
```
Comment (reshamas) [inline blog/blog_gsoc_2022.md:19]:
```suggestion
**Project Name**

Increase Support for Batched Multivariate Distributions

**Mentors**
- [Sayam Kumar](https://github.com/Sayam753)
- [Ricardo Vieira](https://github.com/ricardoV94)
```

Comment (reshamas) [reply blog/blog_gsoc_2022.md:19]: @OriolAbril What do you think of adding the Mentors names? 
I think it would be nice to add their name(s).

**Review by cluhmann — APPROVED** [review]
Looks great to me.

**Comment by reshamas** [comment]
This looks great!  Thanks [@purna135](https://github.com/purna135), @OriolAbril , @cluhmann

**Pull request merged**